http.go

```diff
@@ -32,12 +32,7 @@ func (s *Server) Handler() http.Handler {
 	})
 
 	mux.HandleFuncC(pat.Get("/healthcheck/tracing"), func(c context.Context, w http.ResponseWriter, r *http.Request) {
-		if s.TracingEnabled() {
-			w.Write([]byte("ok\n"))
-		} else {
-			w.WriteHeader(http.StatusForbidden)
-			w.Write([]byte("nok\n"))
-		}
+		w.Write([]byte("ok\n"))
```

Comment:
We should write something in here about deprecating this. Internally we have separate consul services exposed for this — although I don't remember if anything uses them — and we should clean all that up and just say "hey, if there's a Veneur it can trace" as this says.

Reply:
agreed. It'll definitely go on the changelog, do you want it noted anywhere else too?

Reply:
Maybe just a `TODO` here? Otherwise I feel like we're gonna forget all about this. (tbc, we'll forget with the comment here too, but we'll remember. We need like a TODO3.0 or something ;))

Reply:
done the TODO (:

Reply:
I didn't realize this at the time, but your logic triggered it for me. We could change this to be predicated on "is veneur listening for traces?".